src/components/Badge/index.test.js

```diff
@@ -26,14 +26,13 @@ describe('Badge component', () => {
     const notif = container.querySelector(SELECTOR_NOTIF);
     const count = container.querySelector(SELECTOR_COUNT);
     expect(count).toBeFalsy();
+    expect(notif).toBeNull();
+  });
+
+  it('renders correctly as notif without label with showDot true', () => {
+    const { container } = render(<Badge showDot={true} />);
+    const notif = container.querySelector(SELECTOR_NOTIF);
     expect(notif).toBeTruthy();
-    expect(notif).toHaveClass('color--white');
-    expect(notif).toHaveClass('bgColor--red');
-    expect(notif).toHaveClass('shape--circle');
-    expect(notif).toHaveClass('fds-badge--dot');
-    expect(notif).toHaveClass('color--white');
-    expect(notif).not.toHaveClass('color--aqua');
-    expect(notif).not.toHaveClass('bgColor--aqua');
```

Comment:
This is a beautiful kind of explicit that this failed and I had to explicitly fix this test 🔥  A developer can now start to come in and look at the test to figure out what generally is happening.